plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx

```diff
@@ -341,6 +341,20 @@ export const timeSeriesSection = [
         'of query results',
     ),
     controlSetRows: [
+      // eslint-disable-next-line react/jsx-key
+      [<h1 className="section-header">{t('Data imputation')}</h1>],
+      [
+        {
+          name: 'zero_out',
```

Comment:
can the name be more specific e.g. `fill_missing_with_zero` or `fill_zero`? this parameter will be stored.

Reply:
Agreed, will update the name.